### DISCUSSION TITLE
Custom title for 'Select' column

### DISCUSSION THREAD

**Discussion by MatejFacko** [KevinVandy/material-react-table#252] [Q&A] [answered]
Hi,

I am using a row selection and I want to define a custom title/text for Select header. See below screenshot.

![Screenshot 2022-12-13 at 08 28 19](https://user-images.githubusercontent.com/76104834/207253129-23b065e8-d5b8-439e-a045-b639b9b245f7.png)

the default value is 'Select', I want to define specific string.

I tried to add a custom column with accessorKey: 'mrt-row-select' but seems like that the table renders it like a new independent column. Also I wasn't able to find any option how to pass a custom title for Select header. 

`
const columns: ColumnDef<ProgramGrid>[] = [
  {
    // @ts-ignore
    accessorKey: 'mrt-row-select',
    header: 'CUSTOM',
    columnDefType: 'display',
  },
  ....
`

Thank for any advice/suggestions.

**Comment by KevinVandy** [comment, marked as answer]
This is kind of covered in the display columns docs - https://www.material-react-table.com/docs/guides/display-columns

But pretty much you're looking for:

```js
displayColumnDefOptions={{ 'mrt-row-select': { header: 'Your Custom Header' } }}
```

**Comment by MatejFacko** [comment]
@KevinVandy  thanks, works like a charm. Haven't noticed this in docs.
